spec/subjects/async-subject-spec.js

```diff
@@ -1,19 +1,23 @@
 var Rx = require('../../dist/cjs/Rx');
 
 var AsyncSubject = Rx.AsyncSubject;
-var nextTick = Rx.Scheduler.nextTick;
 var Observable = Rx.Observable;
 
 describe('AsyncSubject', function () {
+  var results = [];
```

Comment:
I remember this has been discussed between me and @staltz before, and generally agreed to allow duplicate to let test case to be self-contained. What do you think?